docs/players/Installation_iOS.md

```diff
@@ -4,41 +4,78 @@ You can run VCMI on iOS 12.0 and later, all devices are supported. If you wish t
 
 ## Step 1: Download and install VCMI
 
+The easiest and recommended way to install on a non-jailbroken device is to install the [AltStore Classic](https://altstore.io/) or [Sideloadly](https://sideloadly.io/). We will use AltStore as an example below. Using this method means the VCMI certificate is auto-signed automatically.
+
+i) Use [AltStore Windows](https://faq.altstore.io/altstore-classic/how-to-install-altstore-windows) or [AltStore MacOS](https://faq.altstore.io/altstore-classic/how-to-install-altstore-macos) instructions to install the store depending on the operating system you are using. 
+
+If you're having trouble enabling "sync with this iOS device over Wi-Fi" press on the rectangular shape below "Account". Example shown below.
+
+![image](https://github.com/user-attachments/assets/74fe2ca2-b55c-4b05-b083-89df604248f3)
+
+ii) Download the VCMI-iOS.ipa file on your iOS device directly from the [latest releases](https://github.com/vcmi/vcmi/releases/latest).
+
+iii) To install the .ipa file on your device do one of the following:
+
+•	In AltStore go to >My Apps > press + in the top left corner. Select VCMI-iOS.ipa to install 
+
+or
+
+•	Drag and drop the .ipa file into your iOS device in iTunes
+
+
+## Step 2: Installing Heroes III data files
+
+If you bought HoMM3 on [GOG](https://www.gog.com/de/game/heroes_of_might_and_magic_3_complete_edition), you can download the files directly from the browser in the device. 
+
+Launch VCMI app on the device and the launcher will prompt two files to complete the installation. Select the .bin file first, then the .exe file. This may take a few seconds. Please be patient. 
```

Comment:
```suggestion
Launch VCMI app on the device and the launcher will prompt two files to complete the installation. Select the **.bin** file first, then the **.exe** file. This may take a few seconds. Please be patient. 
```